index.js

```diff
@@ -66,7 +66,7 @@ exports.pack = function (cwd, opts) {
   var ignore = opts.ignore || opts.filter || noop
   var map = opts.map || noop
   var mapStream = opts.mapStream || echo
-  var statNext = statAll(xfs, opts.dereference ? xfs.stat : xfs.lstat, cwd, ignore, opts.entries, opts.sort)
+  var statNext = statAll(xfs, opts.dereference ? 'stat' : 'lstat', cwd, ignore, opts.entries, opts.sort)
```

Comment:
what about using .bind instead? that seems more clean to me

Reply:
I'm not sure, I'm not going to have the time to test such a change as I've moved on from this project atm.

Reply:
I think fs[stat] is more expressive. (and integrate better the the `fs.readdir` spirit)